src/webgl/3d_primitives.js

```diff
@@ -174,8 +174,8 @@ p5.prototype.box = function(width, height, depth, detailX, detailY) {
         [20, 21],
         [22, 23]
       ];
-      for (let i = 0; i < cubeIndices.length; i++) {
-        const cubeIndex = cubeIndices[i];
+
+      cubeIndices.forEach((cubeIndex, i) => {
```

Comment:
Not sure if it is still true or relevant but `forEach` tends to be slower than pure for loops.

Reply:
https://measurethat.net/Benchmarks/Show/2090/0/array-loop-vs-foreach-vs-map#latest_results_block it seems that forEach now is faster.

Reply:
Just tried that benchmark on my computer and for Chrome foreach is faster but for Firefox for loops are about 5-6x faster. A bit surprised Chrome is optimized for foreach though. 🤔
<img width="1184" alt="Screenshot 2023-06-01 at 16 11 30" src="https://github.com/processing/p5.js/assets/7543950/3a4c39a1-3ba8-4389-8c9a-69e584ab43d9">